### PR TITLE
macro to add newly created issues automatically to the Lounge project

### PR DESCRIPTION
**Pull request by roiser**
taken from https://github.com/qmacro/action-add-issue-to-project-column

thanks @hageboeck

### CODE REVIEW

**Review by hageboeck — COMMENTED** [review]
Looking good in principle, but the hard-coded names are maybe not a good idea. 
IIRC, you can give project numbers (2 in this case), because they are stable against renaming.

You get the number from opening the project and checking the URL.

**Comment by hageboeck** [comment]
Can still merge it, though, and test. It won't do harm.

**Comment by roiser** [comment]
makes sense to be safe against renaming (I did already once). So the value of the variable is "project2" ?

**Comment by hageboeck** [comment]
I'm reading up on it. It might just be 'project: 2', but I haven't found it.

**Comment by hageboeck** [comment]
Maybe false alarm. The [javascript code](https://github.com/qmacro/action-add-issue-to-project-column/blob/master/index.js) explicitly retrieves by name. No idea if `2` will count as a valid alias for the actual name.

File: .github/workflows/newissues2lounge.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  assign_issue_to_project:
+    runs-on: ubuntu-latest
+    name: Assign new issue to project
+    steps:
+    - name: Create new project card with issue
+      id: list
+      uses: qmacro/action-add-issue-to-project-column@v1
+      with:
+        token: ${{ secrets.GITHUB_TOKEN }}
+        project: 'Issue Lounge'
```
Comment (hageboeck) [inline .github/workflows/newissues2lounge.yml:15]:
```suggestion
        project: 'Issue_Lounge'
```

Comment (hageboeck) [reply .github/workflows/newissues2lounge.yml:15]: ?

**Review by hageboeck — APPROVED** [review]

File: .github/workflows/newissues2lounge.yml (hunk 1 above)
Comment (roiser) [reply .github/workflows/newissues2lounge.yml:15]: I went the other way round and renamed the project itself

**Pull request merged**